### PR TITLE
[#171433788] Restore mobile-phone item in Profile/Preferences

### PR DESCRIPTION
**Pull request by Edigio**
Restored the visibility for mobile phone number in preferences

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#171433788](https://www.pivotaltracker.com/story/show/171433788): [minor] Ripristinare la visualizzazione del numero di telefono nella sezione Preferenze

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by Edigio** [comment]
Screenshot_1582542381

**Comment by Edigio** [comment]
Screenshot_1582542675

File: ts/screens/profile/PreferencesScreen.tsx [hunk 1]
```diff
@@ -210,7 +212,14 @@ class PreferencesScreen extends React.Component<Props, State> {
               }
               onPress={this.handleEmailOnPress}
             />
-
+            {// Check if spid phone number exists
```
Comment (Undermaken) [inline ts/screens/profile/PreferencesScreen.tsx]: Could you put it below emails (email and SPID email)?
Now the mobile phone is in the between

**Comment by Edigio** [comment]
Set phone-number item below the email items

Simulator Screen Shot - iPhone X - 2020-02-24 at 14 07 30

**Pull request merged**